charts/ingress-nginx/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: ingress-nginx
-version: 2.13.0
+version: 2.13.1
```

Comment:
As this is a new feature we should bump the minor version:
```suggestion
version: 2.14.0
```

Reply:
@ChiefAlexander done, thanks.

Reply:
Sorry, another change got in before yours. Could you bump this again to `2.15.0`?

Reply:
Then squash your changes and this will be good to go

Reply:
No worries @ChiefAlexander. All should be good now.